SMLHelper/BiomeThings/BiomeClass.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using Logger = QModManager.Utility.Logger;
+using UWE;
+using WorldStreaming;
+using System.Runtime.CompilerServices;
+namespace SMLHelper.V2.BiomeThings
```

Comment:
Not the biggest fan of this namespace. Isn't consistent with SMLHelper's naming conventions. Perhaps this should be moved into the `Assets` namespace?

Reply:
Yeah, this could be moved into `SMLHelper.V2.Assets.Biomes`